assets/css/less/front-end/elements/buttons.less

```diff
@@ -19,18 +19,78 @@ body .wpbdp-excerpt {
 		display: inline-flex;
 		align-items: center;
 		justify-content: center;
+		border: 1px solid;
+		border-radius: 4px;
+		padding: 19px 40px;
```

Comment:
This is causing some issues from me in Astra. The button is taller than the inputs. 0 top and bottom padding solves it for me. 
Before:
<img width="1250" alt="Screenshot 2023-08-10 at 5 57 00 PM" src="https://github.com/Strategy11/business-directory-plugin/assets/1116876/1edd5537-7e4f-4344-984b-cae0ca14bf61">

After:
<img width="1255" alt="Screenshot 2023-08-10 at 5 57 16 PM" src="https://github.com/Strategy11/business-directory-plugin/assets/1116876/0140eab1-0533-4b86-b18b-eda741fce128">